src/server/src/main/java/io/cassandrareaper/service/RepairManager.java

```diff
@@ -188,7 +188,8 @@ public RepairRun startRepairRun(RepairRun runToBeStarted) throws ReaperException
         return updatedRun;
       }
       case PAUSED: {
-        RepairRun updatedRun = runToBeStarted.with()
+        RepairRun updatedRun = runToBeStarted
+            .with()
```

Comment:
superfluous changes to the patch. please remove.

Reply:
Why? I just made it readable and similar to NOT_STARTED case. Look at 180 line.

Reply:
it's not related to the PR, and the statement on this line isn't otherwise touched in the PR.
if you'd like to offer code style/formatting improvements that should be done as a separate PR.

Reply:
Why I need to make another PR for this trifle? I don't understand benefit. Can you explain?

Reply:
I mean, I can split it into several commits, isn't enough?